src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -206,15 +207,18 @@ export class SimpleSuggestWidget<TModel extends SimpleCompletionModel<TItem>, TI
 						}
 					}
 
-					const { detail } = item.completion;
+					const { documentation, detail } = item.completion;
+					const docs = strings.format(
+						'{0}{1}',
+						detail || '',
+						documentation ? (typeof documentation === 'string' ? documentation : documentation.value) : '');
 
-					return localize('ariaCurrenttSuggestionReadDetails', '{0}, docs: {1}', label, detail);
+					return localize('ariaCurrenttSuggestionReadDetails', "{0}, docs: {1}", label, docs);
```

Comment:
Move this below the `if (!item.isResolved || !this._isDetailsVisible()) {` block and remove the comment?